### PR TITLE
Redirect to répondeur if no changelog has been deployed

### PR DESCRIPTION
**Pull request by ronnix**

### CODE REVIEW

File: admin/nginx-https.conf.template [hunk 1]
```diff
@@ -26,16 +26,20 @@ server {
...
+    auth_basic "Restricted";
+    auth_basic_user_file /etc/nginx/.htpasswd;
+
+    # Redirect to repondeur if there is no changelog
+    location = / {
+        alias /srv/zam/;
+        try_files index.html /repondeur/;
+    }
+
     location / {
```
Comment (davidbgk) [inline admin/nginx-https.conf.template]: Is that entry still relevant?

Comment (ronnix) [reply admin/nginx-https.conf.template]: There are some let’s encrypt related files there, I supposed we were supposed to serve them. What do you think?

Comment (davidbgk) [reply admin/nginx-https.conf.template]: Oh, I missed the equal sign above, my bad.

Comment (ronnix) [reply admin/nginx-https.conf.template]: No, nginx config format is a bit esoteric...

**Pull request merged**